config/example-menu.cfg

```diff
@@ -3,153 +3,116 @@
 # The snippets in this file may be copied into the main printer.cfg file.
 # See the "example.cfg" file for description of common config parameters.
 
-# Available menu elements:
-#   item - purely visual element
-#   command - same like 'item' but with gcode trigger
-#   input - same like 'command' but has value changing capabilities
-#   list - menu element container, with entry and exit gcode triggers
-#   vsdcard - same as 'list' but will append files from virtual sdcard
+# Menu has default variables defined in menu.cfg
```

Comment:
I don't think we should define default variables in menu.cfg nor have menu specific variables in general.  All of the config parameters are currently available via the `printer.configfile` accessor.  Ultimately, we don't want users to have to configure limits twice (once for the printer and once for the menu).

For the short-term, I'd leave the hardcoded values in place as they are today.  After merging the new menu code, we can then look to see what values can be pulled from the config, and what values need to be added to various `get_status()` methods.  That way users don't have to configure the printer twice.